guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc

```diff
@@ -1,7 +1,12 @@
 [id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
 = Exporting a Content View Version in a Syncable Format
 
-You can export a version of a Content View to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+You can export a version of a Content View to a syncable format that you can use to create your custom CDN.
+After you have exported the Content View, you can do either of the following:
+
+* Synchronize the content from your custom CDN over HTTP/HTTPS.
+* Import the content using `hammer content-import`.
+Note that this requires both the Export and Import servers are the same {ProductVersion} and above.
```

Comment:
This sounds off to me. Please reword.

I am also unsure if "current version or above" make sense in different branches.